schema/show.js

```diff
@@ -315,7 +315,7 @@ const Show = {
       .then(show => {
         if (!show.displayable && !show.is_reference) return new Error('Show Not Found');
         return show;
-      });
+      }).catch(() => null);
```

Comment:
I’m a bit confused about what cases this will catch, doe sit also include the ones returned just above on line 316?